src/OpenLoco/src/Objects/ObjectManager.h

```diff
@@ -194,4 +194,22 @@ namespace OpenLoco::ObjectManager
     void sub_4748FA();
 
     void registerHooks();
+
+    // Calls function with the handle (LoadedObjectHandle) of each loaded object
+    template<typename Function>
+    void forAllLoadedObjects(Function&& func)
```

Comment:
Minor nitpick, I would call it `forEachLoadedObject` which is more in line with a lot of algorithms.